include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3500,6 +3500,7 @@ struct OrtApi {
    *
    * XNNPACK supported keys:
    *   "intra_op_num_threads": number of thread-pool size to use for XNNPACK execution provider.
+   *      default value is 0, which means to use the global thread-pool size.
```

Comment:
Global has a different meaning. Did you mean "session" here?

Reply:
I see. ORT can have multiple session-configs

Reply:
It's because we've the concept of a global threadpool and a session threadpool. Here you mean session since you're passing session options.

Reply:
Got it. No wonder the web side writes it only takes global threadpool.